src/robot/running/testlibraries.py

```diff
@@ -177,6 +177,7 @@ def _create_handlers(self, libcode):
             if method:
                 handler = self._try_to_create_handler(name, method)
                 if handler:
+                    name = getattr(method, 'robot_name', None) or name
                     handlers[name] = handler
```

Comment:
Is second `getattr` really needed here? Wouldn't `handlers[hander.name] = hander` be enough? Would also be safer to have code related to figuring out the name in one place.